### PR TITLE
Add test to prevent usage of strings for credentialSubject.

### PR DESCRIPTION
**Pull request by msporny**
This PR adds a test to ensure that strings are not used for credentialSubject.

### CODE REVIEW

**Review by aljones15 — APPROVED** [review]
Good catch here and thanks p.s. does need linting.

**Comment by brownoxford** [comment]
Should this allow arrays? The data model does indicate that multiple subjects can be provided using an array of objects https://www.w3.org/TR/vc-data-model/#example-specifying-multiple-subjects-in-a-verifiable-credential

**Comment by msporny** [comment]
> Should this allow arrays? The data model does indicate that multiple subjects can be provided using an array of objects https://www.w3.org/TR/vc-data-model/#example-specifying-multiple-subjects-in-a-verifiable-credential

It should allow arrays, but it should not allow empty arrays (which is what this test is testing, IIUC).

File: tests/10-issuer.js [hunk 1]
```diff
@@ -159,7 +159,8 @@ describe('Issue Credential - Data Integrity', function() {
           rowId: this.test.title
         };
         const body = createRequestBody({issuer});
-        const invalidCredentialSubjectTypes = [null, true, 4, []];
+        const invalidCredentialSubjectTypes =
```
Comment (msporny) [inline tests/10-issuer.js:162]:
```suggestion
        const invalidCredentialSubjectTypes =
```

**Pull request merged**